### DISCUSSION TITLE
GitHub Actions are not triggered after recent commit

### DISCUSSION THREAD

**Discussion by melonedo** [community/community#52430] [Actions] [answered]
### Select Topic Area

Bug

### Body

In this PR, https://github.com/openhwgroup/corev-llvm-project/pull/32, there are 3 check in total that should be run. However, after I force-pushed from b8bab04 to e33ab56, two checks stop working and only eclipsefdn/eca is still run, why is this happening?

**Comment by adamdiy** [comment]
getting the same in a private repo today, will update here if we figure it out / GH helps

**Comment by airtower-luna** [comment, marked as answer]
Which workflows do you expect to run? There are quite a few in that repository, and I notice some of them have path filters, so I wonder if that's messing with detecting if a workflow should run. :thinking:
